Azure/communication-ui-library-ios
# [Bugfix] Fix send read receipt

## Purpose

* In the following seniors, a read receipt of the last message should be sent out:
1. When launch the app and joining the chat
2. When re-joining the chat
3. When receiving a new message
4. When manually scrolling  to a new message
5. When jumping to the bottom with the new message button

 These should be tested for both new chat thread and existing chat thread that has more than a screen's messages.

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test


* Open both web and app client
* Send a message from web
* See whether web client's newly sent message show a read receipt icon


## Other Information
